mephisto/client/cli.py

```diff
@@ -27,11 +27,21 @@ def web():
 @cli.command("review")
 @click.argument("review_app_dir", type=click.Path(exists=True))
 @click.option("-p", "--port", type=(int), default=5000)
-def review(review_app_dir, port):
+@click.option("-o", "--output", type=(str), default="")
+@click.option("--stdout", "output_method", flag_value="stdout")
+@click.option("--file", "output_method", flag_value="file", default=True)
+@click.option("--csv-headers/--no-csv-headers", default=False)
+@click.option("-d", "--debug", type=(bool), default=False)
+def review(review_app_dir, port, output, output_method, csv_headers, debug):
     """Transform data from stdin to stdout via a webapp. Good for review."""
```

Comment:
Nit description not fully up to date anymore.

Reply:
good catch, fixed